AndelaOSP/Andela-Socials
# #166286479 Enhance event to handle recurrent events

#### What Does This PR Do?
It enhances event model to handle recurrent events

#### Description Of Task To Be Completed
- Enhance event model to include recurrent start and end date
- Implement bulk create of recurrent events

#### Any Background Context You Want To Provide?
N/A

#### How can this be manually tested?
- Pull the branch
- Run migration by running `python server/manage.py makemigrations` and `python server/manage.py migrate` to update existing model with Recurrent model created
- check the diagram below to see the additional variables to be included to the event model

#### What are the relevant pivotal tracker stories?
[#166286479](https://www.pivotaltracker.com/story/show/166286479)

#### Screenshot(s)
<img width="593" alt="Screenshot 2019-05-31 at 7 53 59 PM" src="https://user-images.githubusercontent.com/28366272/58728036-e1c05480-83dd-11e9-8073-9e3a4e360309.png">
